workflows/dwi_pipe.py

```diff
@@ -646,14 +650,6 @@ def output(self):
         bse= local.path(bse_prefix.split('_desc-')[0]+ '_desc-'+ desc+ '_bse.nii.gz')
         
         
-        # create symlinks
-        if not isfile(dwi):
-            symlink(dwiHcp, dwi)
-            symlink(bvalHcp, bval)
-            symlink(bvecHcp, bvec)
-            symlink(maskHcp, mask)
-            symlink(bseHcp, bse)
```

Comment:
Symlinks are no longer allowed within Samba v3. Hence, we move files to BIDS directory. In the process, we leave an empty file in HCP output directory so that future HCP pipe re-run attempts can skip.